integration_test/manifests/reporting-hub-bop-role-ui/deploy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: reporting-hub-bop-role-ui
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: reporting-hub-bop-role-ui
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: reporting-hub-bop-role-ui
+    spec:
+      containers:
+      - name: app
+        image: ghcr.io/mojaloop/reporting-hub-bop-role-ui:v1.4.1
```

Comment:
What is the reason in defining the K8S manifests manually instead of using the one from helm chart? We can just override the docker image we want in the helm values file.

Reply:
I followed the test harness structure as much as possible, this is just how the ingress was laid out. 
I will revisit the harness in the future.